triton-vm/Cargo.toml

```diff
@@ -30,8 +30,8 @@ version = "1"
 default-features = false
 
 [dependencies]
-triton-opcodes = "0.13"
-twenty-first = "0.11"
+triton-opcodes = { path = "../triton-opcodes" }
+twenty-first = "0.14"
```

Comment:
```suggestion
twenty-first = "0.14.1"
```

Reply:
I'll update the dependencies with the release of the next version Triton VM, after the merge. :+1: